README.md

```diff
@@ -14,9 +14,47 @@ While PVs are well suited for stateful workloads, the Disk objects are aimed tow
 - Predict failures on the Disks, to help with taking preventive actions
 - Allow for dynamically attaching/detaching Disks to a Storage Pod, without requiring a restart
 
-The design and implementation are currently in progress. The design is covered under this [design proposal PR](https://github.com/openebs/node-disk-manager/pull/1)
+The design and implementation are currently in progress. The design is covered under this [design proposal PR](./docs/design.md)
 
 
+# Usage
+## Start Node Disk Manager DaemonSet
+* Edit [ndm-operator.yaml](./ndm-operator.yaml) to fit your environment: Set the `namespace`, `serviceAccount`, configure filters in the `node-disk-manager-config-map`.
+* Switch to Cluster Admin context and create the DaemonSet with `kubectl create -f ndm-operator.yaml`
 
-## License
+## Using `kubectl` to fetch Disk Information
+* `kubectl get disks --show-labels` displays the disks across the cluster, with `kubernetes.io/hostname` showing the node to which disk is attached. 
+* `kubectl get disks -l "kubernetes.io/hostname=<hostname>"` displays the disks attached to node with the provided hostname.
+* `kubectl get disk <disk-cr-name> -o yaml` displays all the details of the disk captured by `ndm` for given disk resource
+
+## Build Image
+* `go get` or `git clone` node-disk-manager repo into `$GOPATH/src/github.com/openebs/`
+with one of the below directions:
+  * `cd $GOPATH/src/github.com/openebs && git clone git@github.com:openebs/node-disk-manager.git`
+  * `cd $GOPATH/src/github.com/openebs && go get github.com/openebs/node-disk-manager`
+
+* Setup build tools:
+  * By default node-disk-manager enables fetching disk attributes using udev. This requires udev develop files. For Ubuntu, `libudev-dev` package should be installed.
+  * `make bootstrap` installs the required go tools
+
+* run `make` in the top directory. It will:
+  * Build the binary.
+  * Build the docker image with the binary.
+
+* Test your changes
+  * `sudo -E env "PATH=$PATH" make test` execute the unit tests
+  * `make integration-test` will launch minikube to run the tests. Make sure that minikube can be executed via `sudo -E minikube start --vm-driver=none`
+
+## Push Image
+By default travis pushes the docker image to `openebs/node-disk-manager`, with ci as well as commit tags. 
```

Comment:
CI

Reply:
ci is correct label. made it highlighted.